main.py

```diff
@@ -226,6 +231,27 @@ def string2timestamp(s):
 def year2timestamp(y):
     return time.mktime(datetime.date(y, 1, 1).timetuple())
 
+@app.route('/newFeatures', methods=['POST'])
+def updatebyemail():
+    emailaddress = str(request.json['address'])
+    if len(emailaddress)>40:
+        return jsonify(respo='Email too long', emailaddress = emailaddress)
+    try:
+        con = sqlite3.connect('emailAddresses.db')
```

Comment:
Why a new database and not the existing one, accessible with the Flask ORM? You can even use the existing User table so no new table has to be created.